hack/packages/package-tools/cmd/package-bundle-generate.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/vmware-tanzu/carvel-imgpkg/pkg/imgpkg/lockconfig"
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/constants"
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/utils"
+)
+
+var packageRepository, version, subVersion string
+var all bool
+
+// packageBundleGenerateCmd is for generating package bundle
+var packageBundleGenerateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate package bundle",
+	RunE:  runPackageBundleGenerate,
+}
+
+func init() {
+	packageBundleCmd.AddCommand(packageBundleGenerateCmd)
+	packageBundleGenerateCmd.Flags().StringVarP(&packageRepository, "repository", "", "", "Package repository of the package bundle being created")
```

Comment:
Change these from `StringVarP` to `StringVar` if no single letter shortcut is going to be provided.